client/analytics/report/customers/table.js

```diff
@@ -128,7 +128,7 @@ export default class CustomersReportTable extends Component {
 			const countryName = this.getCountryName( country );
 
 			const customerNameLink = user_id ? (
-				<Link href={ 'user-edit.php?user_id=' + user_id } type="wp-admin">
+				<Link href={ getAdminLink( 'user-edit.php?user_id=' + user_id ) } type="wp-admin">
```

Comment:
could also be written as (similar to how it's being done elsewhere):

```js
getAdminLink( `user-edit.php?user_id=${ user_id }` )
```

(I think it reads better but totally up to you)